### PR TITLE
[WIP]add document: how to migrate data to tidb

### PR DESCRIPTION
**Pull request by GregoryIan**
### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/how-to/migrate/to-tidb.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: Migrate Data from TiDB to TiDB/MySQL
+summary: migrate data from master TiDB clister to slave TiDB cluster or MySQL.
+category: how-to
+aliases: ['/docs/op-guide/migration/']
```
Comment (ericsyh) [inline v3.0/how-to/migrate/to-tidb.md]:
```suggestion

```
No need to provide aliases for a totally new doc.

File: v3.0/how-to/migrate/to-tidb.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+---
+
+# Migrate Data from TiDB to TiDB/MySQL
+
+## Pre-requirements
+
+* you must already [deploy pump cluster](https://pingcap.com/docs/v3.0/reference/tools/tidb-binlog/deploy/), don't need to deploy drainer.
```
Comment (ericsyh) [inline v3.0/how-to/migrate/to-tidb.md]:
```suggestion
* you must already [deploy pump cluster](/reference/tools/tidb-binlog/deploy/), no need to deploy drainer.
```

File: v3.0/how-to/migrate/to-tidb.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+## Temporal logic of migration opeations
+
+1.  Obtain `initial_commit_ts`. 
+
+2. Use `mydumper` to export data from master TiDB cluster
```
Comment (ericsyh) [inline v3.0/how-to/migrate/to-tidb.md]:
```suggestion
2. Use [mydumper](/reference/tools/mydumper/) to export data from master TiDB cluster
```
Link to the mydumper instructions doc.

File: v3.0/how-to/migrate/to-tidb.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+3. Use `loader` to import the data into slave TiDB cluster or MySQL.
```
Comment (ericsyh) [inline v3.0/how-to/migrate/to-tidb.md]:
```suggestion
3. Use [loader](/reference/tools/loader.md) to import the data into slave TiDB cluster or MySQL.
```

**Comment by kolbe** [comment]
What is the actual purpose of this document? The filename is "to-tidb.md" but the title is "Migrate Data from TiDB to TiDB/MySQL" ... "to" v. "from".

It looks like this document is mostly some kind of tutorial for how to use TiDB-Binlog. Is that right?

**Comment by lilin90** [comment]
> What is the actual purpose of this document? The filename is "to-tidb.md" but the title is "Migrate Data from TiDB to TiDB/MySQL" ... "to" v. "from".
> 
> It looks like this document is mostly some kind of tutorial for how to use TiDB-Binlog. Is that right?

- @GregoryIan I suggest updating the filename to `from-tidb.md`.
- @kolbe IMO, this file is more task-oriented.

**Comment by lilin90** [comment]
@GregoryIan Are you still working on updating this document?

**Comment by GregoryIan** [comment]
I will close it, thanks everyone

**Pull request closed without merging**